test/hotspot/jtreg/containers/docker/TestPids.java

```diff
@@ -93,6 +93,10 @@ private static void checkResult(List<String> lines, String lineMarker, String ex
                 String[] parts = line.split(":");
                 System.out.println("DEBUG: line = " + line);
                 System.out.println("DEBUG: parts.length = " + parts.length);
+                if (expectedValue.equals("no_value_expected")) {
+                    System.out.println("No value expected for " + lineMarker);
```

Comment:
Perhaps this debug print could say `System.out.println("Found '" + lineMarker + "' with value: '" + <value> +"');` and actually parse the number as we'd expect an integer there?